src/Widgets/DataEmptyWidget/DataEmptyWidgetEditingConfig.ts

```diff
@@ -0,0 +1,12 @@
+import { provideEditingConfig } from 'scrivito'
+import { DataEmptyWidget } from './DataEmptyWidgetClass'
+import Thumbnail from './thumbnail.svg'
+import { TextWidget } from '../TextWidget/TextWidgetClass'
+
+provideEditingConfig(DataEmptyWidget, {
+  title: 'Data Empty',
```

Comment:
* [ ] FYI It would be great to have a more self-explanatory title. Sth. like "If no data", "Missing data fallback", "Fallback".